lib/supabaseClient.ts

```diff
@@ -1,8 +1,8 @@
 import { createClient } from "@supabase/supabase-js";
 
 export const supabase = createClient(
-  process.env.NODE_ENV === "development"
-    ? "http://localhost:54321"
-    : `https://${process.env.SUPABASE_PROJECT_ID!}.supabase.co`,
+  process.env.SUPABASE_API_URL!
```

Comment:
can be shortened as `process.env.SUPABASE_API_URL || "http://localhost:54321"`

Reply:
Makes sense, pushed it.